runtime/src/accounts_db.rs

```diff
@@ -651,6 +796,7 @@ impl AccountsDB {
                             &mut reclaims,
                             max_clean_root,
                             &self.account_indexes,
+                            |account_info| account_info.store_id != CACHE_VIRTUAL_STORAGE_ID,
```

Comment:
well, this is a closure given to `is_cache`. but the conditions are inverted...

how about like this?:

```rust
/// here
|account_info| account_info.store_id == CACHE_VIRTUAL_STORAGE_ID

...
/// in accounts index
let should_purge = Self::can_purge(max_root, *slot) && !is_cached(value)
```